_posts/2024-04-02-shared-conda-tutorial.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: An ideal setup for a multi-user conda installation
+tags:
+  - conda
+  - bash
+  - systems-administration
+contributors:
+- boulund
+- pmenzel
+- hexylena
```

Comment:
```suggestion
- hexylena
reviewers:
- karinlag
- bebatut
```

Reply:
I think in the future it could be interesting to automate this, have a github workflow that responds to [pull request reviews](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review) and automatically suggests adding their name to the PR.